website/pages/docs/glossary/what-is-the-modern-data-stack.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: "What is The Modern Data Stack?"
+tag: engineering
+date: 2023/03/16
+description: >-
+  Past, Present and future of the modern data stack
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../../components/BlogHeader"
+
+<BlogHeader/>
```

Comment:
```suggestion
```

Maybe we should drop this section? The blog header doesn't seem to fit well in here:
![image](https://user-images.githubusercontent.com/26760571/227770927-b8dd6c7c-5f1a-4d52-bb43-533957abafb3.png)